common-utils/src/main/java/eu/arrowhead/common/intf/properties/validators/NotEmptySetValidator.java

```diff
@@ -0,0 +1,70 @@
+package eu.arrowhead.common.intf.properties.validators;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import eu.arrowhead.common.Utilities;
+import eu.arrowhead.common.exception.InvalidParameterException;
+import eu.arrowhead.common.intf.properties.IPropertyValidator;
+import eu.arrowhead.common.service.validation.name.NameNormalizer;
+import eu.arrowhead.common.service.validation.name.NameValidator;
+
+@Service
+public class NotEmptySetValidator implements IPropertyValidator {
```

Comment:
I think this is a wrong name for the class (and the enum as well). This one only works with string sets. NotEmptyStringSetValidator?